### DISCUSSION TITLE
When PrimeNG 18 preview would be released?

### DISCUSSION THREAD

**Discussion by ProximaB** [primefaces/community#2286] [PrimeNG] [answered]
I'm starting a new project and I've been recommended to use PrimeNg, but I don't want to; invest time to upgrade to 18 in a few months.
Is there an option to release a preview to help start with version 18 and increase adoption of the new version. 
It would be great to have documentation as well :)

**Comment by NicolasAssmann** [comment]
Hello,

I am in the same situation. I need to start a new WebApp using Angular 18. I would like to use PrimeNG just like our last WebApp, but I have the same dilemma:
Either version 18 is released in the very near future, and we can start this development at that time.
Or version 18 is not scheduled for summer release, and we cannot wait for its release to match our own release.

If you have any indication, even approximate, to help teams facing this dilemma to position themselves, it would be really nice.

Thank you.

**Comment by ProximaB** [comment]
Could you please consider posting a comment? This situation suggests that PrimeNG may not be maintained as well as it could be. 
It might be beneficial to expand your team if managing both 'public relations' and code development is becoming challenging. 

Material Design for Angular seems to have an advantage in this aspect, as they have already released a version for Angular 18 that includes CSS variables and a functional designer.

**Comment by csgijohan** [comment]
Have you seen this post? 
https://www.primefaces.org/blog/primeng-for-angular-18/
E.g. 
> PrimeNG v17.18.0 is available for compatibility with Angular v18

So you can use the current version of PrimeNg **v17.18.x** with Angular **v18**

I agree that an official update here would be good.

**Reply by NicolasAssmann** [reply]
Hi ! 

I saw that but it's mostly a news to say "we updated dependencies so now you can run it with angular 18". Which is not a solid : " it works well with angular18"
Because indeed a lot of things are not working that well on angular18 (animation). 
OR it need some customisation that are not documented

**Comment by qwadrox** [comment, marked as answer]
@ProximaB The beta has been released: https://v18.primeng.org/

**Comment by omar-alsaheb** [comment]
When will v18 be latest and stable NOT Beta ??

**Comment by LMatass** [comment]
Any news here? When will primeng V18 be out of beta?

**Comment by giuseppebrb** [comment]
It would be nice to have also updates on a release date for Material theme in PrimeNG v18
